### PR TITLE
Added sign in with google and git hub and theme integration.

### PR DESCRIPTION
**Pull request by bacancy-urmit**
Added sign in with google and git hub and theme integration.

### CODE REVIEW

File: app/views/companies/index.html.erb [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-    Hello <%= current_company.name %><br/>
-    <%= link_to 'Jobs', company_opening_jobs_path(current_company) %><br/>
-  <% else %>
-    <%= link_to 'Logout', destroy_company_session_path, method: :delete %><br/> 
-    <center>
-      <h2>Your trial period Expired!</h2>
-      <h5 style="color: red">(You can activate your account <a style="color: green" href="/companies/<%= current_company.id %>/charges/new">here</a>)</h5>
-    </center>
-  <% end %>
+<div class="main-content">
```
Comment (swati-bacancy) [inline app/views/companies/index.html.erb:2]: indentation

File: app/views/layouts/_user_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </header>
+      <!-- HEADER DESKTOP-->
+
```
Comment (swati-bacancy) [inline app/views/layouts/_user_header.html.erb:86]: EOL

Comment (bacancy-urmit) [reply app/views/layouts/_user_header.html.erb:86]: EOL is already added and if you are asking about incomplete tags then we completed it in another page as we have header and side bar in our head bar.

File: app/views/companies/index.html.erb (hunk 1 above)
Comment (bacancy-urmit) [reply app/views/companies/index.html.erb:2]: applied proper indentation.

**Pull request merged**